### DISCUSSION TITLE
How can I use the new API wrappers for interactions?

### DISCUSSION THREAD

**Discussion by blnayan** [discordjs/discord.js#5871] [Q&A] [answered]
In the homepage of [discord.js.org](discord.js.org) there is a snippet of code that shows how to use the new API wrappers for interactions.
![image](https://user-images.githubusercontent.com/52083057/122504796-01c50f80-cfc9-11eb-8fb8-54180f7d2bea.png)
When downloading discord.js library this wrapper is not usable since it downloaded the stable version which doesn't include wrappers for interactions. Is there any way to use the wrappers for interactions?

**Comment by olivermrose** [comment, marked as answer]
You can download the development branch with `npm install discord.js@dev`.

**Resources:**
- [Documentation](https://discord.js.org/#/docs/main/master/general/welcome)
- [Migration Guide](https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html)
- [Slash Commands Guide](https://deploy-preview-638--discordjs-guide.netlify.app/interactions/registering-slash-commands.html)
- [Buttons Guide](https://deploy-preview-674--discordjs-guide.netlify.app/interactions/buttons.html)

**Reply by blnayan** [reply]
Do you know when will v13 will be released?

**Reply by almostSouji** [reply]
> v13 release: mid-end of June btw.

If you want to receive announcements around discord.js you can follow our announcement channel at https://discord.gg/djs
